packages/extension-redis/src/Redis.ts

```diff
@@ -310,6 +310,17 @@ export class Redis implements Extension {
    * if the ydoc changed, we'll need to inform other Hocuspocus servers about it.
    */
   public async onChange(data: onChangePayload): Promise<any> {
+    // Check if there is only one client connected (`clientsCount <= 1`), the transaction
+    // originates from the system itself (no `transactionOrigin`), and there is no associated
+    // socket ID. This indicates that the change was triggered by the server itself.
+    if (data.clientsCount <= 1 && !data.transactionOrigin && !data.socketId) {
```

Comment:
not sure why this is needed, because we're anyway sending all updates below (L324) 🤔